vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -116,7 +121,7 @@ function Get-FilesToCacheOnVHD {
 
     $map = @{
         "c:\akse-cache\"              = @(
-            "https://github.com/Azure/AgentBaker/raw/master/vhdbuilder/scripts/windows/collect-windows-logs.ps1",
+            "https://github.com/Azure/AgentBaker/raw/update-k8s-hotfixes/vhdbuilder/scripts/windows/collect-windows-logs.ps1",
```

Comment:
update-k8s-hotfixes => master?
How about to check in collect-windows-logs.ps1 at first without running tests? Otherwise we may forget to update this.

Reply:
Good idea.

Reply:
Here it is https://github.com/Azure/AgentBaker/pull/704